testing/tests/DevExpress.ui.widgets.pivotGrid/pivotGrid.markup.tests.js

```diff
@@ -0,0 +1,56 @@
+"use strict";
```

Comment:
Is it possible to reuse some of existing PivotGrid tests?

Reply:
PivotGrid with SSR creates empty PivotGrid.
I've examined tests and not found tests for empty markup. All of them with data and check functionality.

Reply:
Oh, right.